evaluator/evaluate.go

```diff
@@ -206,15 +206,15 @@ func tryGetYumUpdates(system *models.SystemPlatform) (*vmaas.UpdatesV2Response,
 }
 
 func evaluateWithVmaas(tx *gorm.DB, updatesData *vmaas.UpdatesV2Response,
-	system *models.SystemPlatform, accountName string) (*vmaas.UpdatesV2Response, error) {
+	system *models.SystemPlatform, accountName, orgID string) (*vmaas.UpdatesV2Response, error) {
 	if enableBaselineEval {
 		err := limitVmaasToBaseline(tx, system, updatesData)
 		if err != nil {
 			return nil, errors.Wrap(err, "Failed to evaluate baseline")
 		}
 	}
 
-	err := evaluateAndStore(tx, system, updatesData, accountName)
+	err := evaluateAndStore(tx, system, updatesData, accountName, orgID)
 	if err != nil {
```

Comment:
And here actually `accountName, orgID` is the `event.GetAccountName(), event.GetOrgID()` from above. And is passed to `evaluateAndStore(tx, system, updatesData, accountName, orgID)` which could be changed to `evaluateAndStore(tx, system, updatesData, event)` as well.